.ci/run_metrics_PR_ci.sh

```diff
@@ -67,7 +73,30 @@ check() {
 		sudo make install
 		popd
 
-		checkmetrics --percentage --basefile /etc/checkmetrics/checkmetrics-json-$(uname -n).toml --metricsdir ${RESULTS_DIR}
+		# FIXME - we need to document the whole metrics CI setup, and the ways it
+		# can be configured and adapted. See:
+		# https://github.com/kata-containers/ci/issues/58
+		#
+		# If we are running on a (transient) cloud machine then we cannot
+		# tie the name of the expected results config file to the machine name - but
+		# we can expect the cloud image to have a default named file in the correct
+		# place.
+		if [ -n "${METRICS_CI_CLOUD}" ]; then
+			local CM_BASE_FILE="${CHECKMETRICS_CONFIG_DIR}/checkmetrics-json.toml"
```

Comment:
Amusing filenames ftw! :smile:

Reply:
Yep - whilst writing this PR, when I ran into that it made me 👀 squint somewhat and scratch my head - and I wrote that file!!! Finally I figured out it is named that way as it is a toml file that indexes a bunch of json files - whereas its predecessor used to index csv files... I'm not going to rename it now unless somebody really wants me to ;-)